### ISSUE TITLE
Add Nest Hello as doorbell 

### ISSUE THREAD

**Issue by hyperbart** [KraigM/homebridge-nest#188]
Nest Hello just got released in Belgium, would be nice to integrate the Hello in Homekit by means of this plugin.

**Comment by chrisjshull** [comment]
This is no longer the canonical repo for this module. Please file a bug here: https://github.com/chrisjshull/homebridge-nest/issues/new/choose

However, I do not see an API for the Hello yet: https://developers.nest.com/documentation/cloud/get-started

**Comment by mbnn** [comment]
Yes it does: https://nestdevelopers.io/t/nest-doorbells-api/1039/3
Shows up same as other nest camera's :)

**Comment by Sibelle65** [comment]
> Yes it does: https://nestdevelopers.io/t/nest-doorbells-api/1039/3
> Shows up same as other nest camera's :)

this link does not show any api details of the Hello doorbell....

**Comment by qthuan** [comment]
> > Yes it does: https://nestdevelopers.io/t/nest-doorbells-api/1039/3
> > Shows up same as other nest camera's :)
> 
> this link does not show any api details of the Hello doorbell....

This have API for Nest Hello, it's camera API:
https://developers.nest.com/guides/api/camera-guide#camera_types

**Comment by Sibelle65** [comment]
> > > Yes it does: https://nestdevelopers.io/t/nest-doorbells-api/1039/3
> > > Shows up same as other nest camera's :)
> > 
> > 
> > this link does not show any api details of the Hello doorbell....
> 
> This have API for Nest Hello, it's camera API:
> https://developers.nest.com/guides/api/camera-guide#camera_types

Thanx for your reply... Only the Hello camera function is included, not the doorbell function/speaker. So that does not work for me as an alternative. I have to wait some more time I guess.
